app/src/main/res/values-it/strings.xml

```diff
@@ -26,16 +33,14 @@
     <string name="gotorating">Portami!</string>
     <string name="nevershowagain">Lasciami stare!</string>
     <string name="reconsider">Forse dopo</string>
-    <string name="donation_message">Sembra che Substratum Extras for Android Auto ti piaccia! Vorresti aiutarmi facendo una piccola donazione?</string>
-    <string name="helpButton">🆘 Aiuto 🆘</string>
+    <string name="donation_message">Sembra che Substratum Extras for Android Auto ti piaccia!\nVorresti aiutarmi facendo una piccola donazione?</string>
     <string name="copied_bitcoin">Indirizzo Bitcoin copiato negli appunti</string>
     <string name="helpinstalling">Ho bisogno di aiuto per installare</string>
-    <string name="helpme">Per installare un extra:
-        \n1. Guarda il messaggio di attenzione (simbolo di attenzione accanto ad Android Auto) per evitare errori
-        \n2. Seleziona Android Auto e poi un extra dalla lista
-        \n3. Premi il bottone in basso e seleziona \'Costruisci e installa\'
-        \n4. Ripetere i passi 2 e 3 a proprio piacimento
-        \n3. Riavvia il dispositivo, ri-apri Substratum, recati alla sezione gestore, e abilita gli extra</string>
+    <string name="faqstring" translatable="false">FAQ</string>
```

Comment:
No need to translate untraslatable strings

Reply:
In Italian could be "Domande frequenti (FAQ)"